pkg/cmd/server/origin/openshift_apiserver.go

```diff
@@ -342,12 +342,12 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	initOpenshiftVersionRoute(s.GenericAPIServer.Handler.GoRestfulContainer, "/version/openshift")
 
 	// register our poststarthooks
-	s.GenericAPIServer.AddPostStartHook("quota.openshift.io-clusterquotamapping", c.startClusterQuotaMapping)
-	s.GenericAPIServer.AddPostStartHook("project.openshift.io-projectcache", c.startProjectCache)
-	s.GenericAPIServer.AddPostStartHook("project.openshift.io-projectauthorizationcache", c.startProjectAuthorizationCache)
-	s.GenericAPIServer.AddPostStartHook("security.openshift.io-bootstrapscc", c.bootstrapSCC)
-	s.GenericAPIServer.AddPostStartHook("authorization.openshift.io-ensureSARolesDefault", c.ensureDefaultNamespaceServiceAccountRoles)
-	s.GenericAPIServer.AddPostStartHook("authorization.openshift.io-ensureopenshift-infra", c.ensureOpenShiftInfraNamespace)
+	ensureAddPostStartHook(s.GenericAPIServer, "quota.openshift.io-clusterquotamapping", c.startClusterQuotaMapping)
+	ensureAddPostStartHook(s.GenericAPIServer, "project.openshift.io-projectcache", c.startProjectCache)
+	ensureAddPostStartHook(s.GenericAPIServer, "project.openshift.io-projectauthorizationcache", c.startProjectAuthorizationCache)
+	ensureAddPostStartHook(s.GenericAPIServer, "security.openshift.io-bootstrapscc", c.bootstrapSCC)
+	ensureAddPostStartHook(s.GenericAPIServer, "authorization.openshift.io-ensureSARolesDefault", c.ensureDefaultNamespaceServiceAccountRoles)
```

Comment:
Sorry fail to see the inconsistency, ensure is lower case because it is a private function.